ocs_ci/framework/__init__.py

```diff
@@ -273,6 +273,19 @@ def get_provider_index(self):
 
         raise ClusterNotFoundException("Didn't find the provider cluster")
 
+    def get_provider_cluster_indexes(self):
```

Comment:
We have `get_provider_index()` function. Do we expect to have multiple providers in one test run?

Reply:
Just added this capability. Multiple RDR clusters on ACM hub is a feature. If we need to use this , this will be useful